src/test/java/com/exasol/errorreporting/PlaceholderTest.java

```diff
@@ -0,0 +1,67 @@
+package com.exasol.errorreporting;
+
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+class PlaceholderTest {
+    @Test
+    void TestParseParameterNameWithoutSwitches()
+    {
+        final Placeholder placeholder = Placeholder.parse("the_parameter").build();
+        assertAll(
+                () -> assertThat(placeholder.getReference(), equalTo("the_parameter")),
+                () -> assertThat(placeholder.getQuoting(), equalTo(Quoting.AUTOMATIC))
```

Comment:
This is duplicated in all tests. Maybe extract this into a method `assertParameter()`?

Reply:
Done.